package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "@demos-europe/demosplan-ui",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "license": "MIT",
   "private": false,
   "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
   "dependencies": {
     "@braintree/sanitize-url": "^6.0.1",
+    "@demos-europe/demosplan-ui": "^0.0.2",
```

Comment:
Why does this package have itself as a dependency now?

Reply:
Because even before the package was required. But I just didn't checked why, but only recognized it. Stupid thing though. Changed it in 0a132519c3a9037d28a1a42f88e5c57f3896d2f1